README.md

```diff
@@ -147,13 +161,24 @@ jobs:
           token: ${{ secrets.RENOVATE_TOKEN }}
 ```
 
+We recommend you pin the version of Renovate to a full version or a full checksum, and use Renovate's regex manager to create PRs to update the pinned version.
+We created a [Regex Manager Preset, UPDATE TEXT AND LINK TO POINT TO CORRECT LOCATION](https://docs.renovatebot.com/presets-regexManagers/) to help you.
+
+```json
+{
+  "extends": ["regexManagers:AAAAAAAAA"]
+}
+```
```

Comment:
> we probably should suggest pin and provide a regex manager to auto update as preset

I like this idea, here's a really rough draft for it. 😄 

Todo:

- [ ] Decide on pinning to just a `v1.2.3` version, or to a full checksum
- [ ] Create a Regex Manager Preset to help users update their GitHub Action Renovate version
- [ ] Update the link text and link location
- [ ] Update the config example so it actually works

Reply:
can we move this to a separate PR?

Reply:
Good idea, I'll move my todo list to a separate issue. 😄